src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -317,30 +317,11 @@ final class ExposureNotificationSettingViewController: UITableViewController, Ac
 	}
 
 	private func tracingCell(for indexPath: IndexPath, in tableView: UITableView) -> UITableViewCell {
```

Comment:
We could refactor this as it is not tracingCell anymore.

Reply:
Please see above.